.github/readmes/javascript/rest-express/README.md

```diff
@@ -2,9 +2,13 @@
 
 This example shows how to implement a **REST API** using [Express](https://expressjs.com/) and [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client). It uses a SQLite database file with some initial dummy data which you can find at [`./prisma/dev.db`](./prisma/dev.db).
 
+__INLINE(../_setup-0.md)__
+curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=2 prisma-examples-latest/javascript/rest-express`
```

Comment:
The ` at the end is probably wrong here?

Reply:
Thanks for catching that. I must have missed that